### ISSUE TITLE
please delete open rst in setup.py

### ISSUE THREAD

**Issue by rendiya** [taraslayshchuk/es2csv#47]
Collecting es2csv
  Downloading https://files.pythonhosted.org/packages/3d/3a/6dedac602d4b10022d427a78e9b718a57a7254e02810906b99b41a6e6c72/es2csv-5.2.1.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-build-c9kw1uju/es2csv/setup.py", line 26, in <module>
        with open('HISTORY.rst') as history_file:
    FileNotFoundError: [Errno 2] No such file or directory: 'HISTORY.rst'
    
this error because pypi cannot save .rst file
thank you

**Comment by taraslayshchuk** [comment]
It has been fixed in 5.5.2.

**Comment by slhck** [comment]
The issue is with Python 3, but that's not apparent to the user.

```
➜ pip install es2csv
Collecting es2csv
  Downloading https://files.pythonhosted.org/packages/3d/3a/6dedac602d4b10022d427a78e9b718a57a7254e02810906b99b41a6e6c72/es2csv-5.2.1.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/private/var/folders/1s/9jbdrzg17wvcnr76p_ntdnbw0000gn/T/pip-build-4sh53n4v/es2csv/setup.py", line 26, in <module>
        with open('HISTORY.rst') as history_file:
    FileNotFoundError: [Errno 2] No such file or directory: 'HISTORY.rst'

    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /private/var/folders/1s/9jbdrzg17wvcnr76p_ntdnbw0000gn/T/pip-build-4sh53n4v/es2csv/
```

When using Python 3, the oldest version of the package without specific Python requirements is installed. Perhaps it makes sense to retroactively change these requirements for old releases, too? Or is that not possible with PyPI? (Not a big expert when it comes to that.)
